docs/en/observability/threshold-alert.asciidoc

```diff
@@ -151,4 +151,4 @@ You can add more context to the message by clicking the icon above the message t
 and selecting from a list of available variables.
 
 [role="screenshot"]
-image::images/logs-threshold-alert-default-message.png[Default notification message for logs threshold rules with open "Add variable" popup listing available action variables,width=600]
\ No newline at end of file
+image::images/logs-threshold-alert-default-message.png[Default notification message for logs threshold rules with open "Add variable" popup listing available action variables,width=600]
```

Comment:
Looks like this screenshot still has Technical Preview in the heading.

![image](https://github.com/elastic/observability-docs/assets/114418652/1ea25eff-f9be-4e0d-9399-6a3316bdf4df)

Reply:
sorry, not the screenshot in line 154, but the one in line 13

Reply:
good spot. done!